src/ViewModels/WebMapServiceItemViewModel.js

```diff
@@ -108,8 +108,8 @@ var WebMapServiceItemViewModel = function(application) {
     delete this.__knockoutObservables.dataUrlType;
     knockout.defineProperty(this, 'dataUrlType', {
         get : function() {
-            if (defined(this._dataUrlType)) {
-                return this._dataUrlType;
+            if (defined(this._dataUrl)) {
+                return 'direct';
```

Comment:
We had only one of each case in the initialization file that were resolved away pretty quickly so it is currently not necessary.  We can discuss if we need the extra functionality.  The best way to do this going forward is probably to get out of this business altogether and point all the data set links to ckan pages and let people do what they want there.  That has a lot more flexibility than we will ever probably want to have.